src/icons/icons.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Title, IconGallery, IconItem } from "@storybook/blocks";
+
+import * as Icons from ".";
+
+<Meta title="Icons gallery" />
+
+# Icons Gallery
+
+List of available icons
+
+<IconGallery>
```

Comment:
I think you can also do this with a normal story file, what advantage are we getting for mdx?
Curious about it, I don't have a lot of experience with mdx

Reply:
The `<IconGallery>` component doesn't work if it's not used inside a mdx file. In addition to that, Icon gallery is not technically a story, I did it like that before this (creating my own `IconGallery` component, and so on), and I didn't like the result.

Reply:
Makes sense. Thanks for the clarification!